File: common/library-old/src/main/java/com/adair/core/http/HttpConstant.java
package com.adair.core.http;

/**
 * @author XuShuai
 * @version v1.0
 * @date 2021/4/6
 */
public class HttpConstant {
    public static final Long TIMEOUT_CONNECT = 15L;
    public static final Long TIMEOUT_READ = 30L;
    public static final Long TIMEOUT_WRITE = 30L;
    public static final Long CACHE_SIZE = 50L * 1024L * 1024L; //缓存大小50M

    /**
     * 当接口需要其他url时，配置header, value 为 新的url路径
     */
    public static final String NEW_URL_HEADER = "newUrl";
    public static final String LOG_TAG = "library_network";
}

File: common/lib-http/src/main/java/com/adair/httpv2/HttpConstant.kt
package com.adair.httpv2

/**
 *
 * 静态属性配置
 *
 * @author xushuai
 * @date   2021/11/10-22:33
 * @email  466911254@qq.com
 */

//连接超时
const val TIMEOUT_CONNECT: Long = 15L
//读取超时
const val TIMEOUT_READ = 30L
//写入超时
const val TIMEOUT_WRITE = 30L
//缓存最大占用
const val CACHE_SIZE = 50L * 1024L * 1024L //缓存大小50M

/**
 * 当接口需要其他url时，配置header, value 为 新的url路径
 */
const val NEW_URL_HEADER = "newUrl"
const val LOG_TAG = "library_network"